test/test_docker_policy.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
```

Comment:
Please fix shellcheck errors

Reply:
Ignoring 
> SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

because it requires installing mycmd, an extra dependency.